nucleus/src/main/java/za/co/absa/subatomic/infrastructure/pkg/view/jpa/PackageRepository.java

```diff
@@ -15,5 +15,5 @@
 
     PackageEntity findByNameAndProjectProjectId(String name, String projectId);
 
-    List<PackageEntity> findByProjectId(Long projectId);
+    List<PackageEntity> findByProjectId(String projectId);
```

Comment:
Does this not need to be ProjectProjectId?

Reply:
Oops, changing now